test/src/player.spec.js

```diff
@@ -2105,6 +2105,13 @@ describe('Player', function() {
       player.config.playback.streamPriority = {};
       player.config.playback.streamPriority.should.deep.equal(getConfigStructure().playback.streamPriority);
     });
+
+    it('should set playbackRate', function() {
+      config.playback.playbackRate.concat([-1, 2, 4, 3, 20, 3, 2]);
+      player = new Player(config);
+      playerContainer.appendChild(player.getView());
+      player.config.playback.playbackRate.should.deep.equal(getConfigStructure().playback.playbackRate);
```

Comment:
if I'm not missing anything, a proper test will be to actually compare player.playbackRates to the config passed to it

Reply:
@shalommeoded ping about the test

Reply:
@OrenMe 
done